src/appcast.cpp

```diff
@@ -208,7 +228,7 @@ void XMLCALL OnEndElement(void *data, const char *name)
 
     if ( ctxt.in_item && strcmp(name, NODE_RELNOTES) == 0 )
     {
-        ctxt.in_relnotes--;
+        ctxt.in_relnotes && ctxt.in_relnotes--;
```

Comment:
Please don’t use oh-so-clever hacks like this, they only obfuscate the meaning and make it hard to read the code. Use `if` for branching in statements.